src/components/activities/ActivityCard.jsx

```diff
@@ -234,14 +281,14 @@ class ActivityCard extends Component {
           <div className='activity__content'>
             <TruncateDescription description={description} wordCount={wordCount} />
           </div>
-          <div className='activity__footer'>
-            {
-              this.renderLocationOrPoints()
-            }
-            {
-              this.renderButtonsOrStatus()
-            }
-          </div>
+        </div>
+        <div className='activity__footer'>
```

Comment:
This is more of a best practice thing. Good job on delegating this functionality but what  is your opinion of calling the function before the return and then placing the necessary jsx as a variable within the return function? 

To ensure we keep the return statement as clean as possible.

Reply:
I don't really have strong inclinations either way. It's just a pattern that I'm used to. The JSX would look cleaner though, so I'll switch to the best practice